### DISCUSSION TITLE
bulb-off2 and air-conditioning-off for home automation?

### DISCUSSION THREAD

**Discussion by davekc** [tabler/tabler-icons#338] [Ideas]
What about a bulb-off2?
![bulb-off2](https://user-images.githubusercontent.com/11339519/196064486-2038f894-d907-4456-9875-0d452141daaa.svg)

And air-conditioning-off?
![air-conditioning-off](https://user-images.githubusercontent.com/11339519/196064668-a4018cca-def8-4a4e-8999-46f39e79da1f.svg)
Really usefull icons, thanks a lot for your work!

**Comment by codecalm** [comment]
I've added `air-conditioning-disabled` in https://github.com/tabler/tabler-icons/releases/tag/v1.105.0 :) 

![tabler-icons-1 105 0](https://user-images.githubusercontent.com/1282324/196275553-16439368-a909-44e4-b23b-e4e195feef2d.png)

**Reply by davekc** [reply]
Great, that was quick ;) Thanks a lot! Do you consider the bulb as well?

**Reply by davekc** [reply]
I modified the bulb myself, so you don’t have to worry about that. Thanks!

**Comment by davekc** [comment]
I just realized that the air-conditioning-disabled version center itself vertically, which seems logical, but the air-conditioning version cannot sit at the same place (to achieve an on/off state without having the icons moving around). Maybe I should implement my own modified versions, especially since I already use the SVG approach. That way I can save you a lot of unnecessary hassle.

**Reply by davekc** [reply]
Finally, I change my mind, I think you were right to center the icon. Yes, it moves, but remains centered so aesthetic no matter the state. Thanks again!
